src/main/java/org/teamresistance/frc/Snorfle.java

```diff
@@ -0,0 +1,34 @@
+package org.teamresistance.frc;
+
+import org.strongback.command.Command;
+import org.strongback.components.Motor;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Snorfle extends Command {
```

Comment:
Now is also a good time to start packaging things up―could you move the Snorfler commands into a dedicated package? You may also want to then consider making this class `protected` to keep it self-contained.